utils/build_pydusa_numpy.sh

```diff
@@ -17,12 +17,17 @@ set -xe
 
 RECIPES_DIR=$(cd $(dirname $0)/../recipes && pwd -P)
 
-if [ ${#args} -lt 1 ];then
+number_of_words=$(wc -w <<< "${args[@]}")
```

Comment:
If this is to get number of arguments, `${#args}` gives exactly that without the need for a subshell, `$(...)`.